wpiutil/src/main/native/include/wpi/StringMap.h

```diff
@@ -36,7 +36,6 @@ namespace wpi {
 template<typename ValueTy> class StringMapConstIterator;
 template<typename ValueTy> class StringMapIterator;
 template<typename ValueTy> class StringMapKeyIterator;
-template<typename ValueTy> class StringMapEntry;
 
```

Comment:
Yes, a line was removed here (forward declaration of StringMapEntry).  This is an llvm file so let's undo the changes here.